### DISCUSSION TITLE
Unable to use Style in xaml

### DISCUSSION THREAD

**Discussion by ObsidianSilence** [canton7/Stylet#233] [Q&A]
This is my first project using Stylet so I am probably missing something. I am opening a number of views that open other views, all of the views have a button that opens another view and a GoBack button. When I have all of the styling information in each button the project compiles and runs just fine.
When I pull the styling values out into Styles.xaml and use the Style tag
![image](https://user-images.githubusercontent.com/50368420/111194107-3c19ad80-8591-11eb-9bb9-9f0993090180.png)
![image](https://user-images.githubusercontent.com/50368420/111194176-4b98f680-8591-11eb-9619-b19ecc878823.png)
I get the following error message
"{DependencyProperty.UnsetValue}" is not a valid value for property 'Foreground'.
![image](https://user-images.githubusercontent.com/50368420/111194234-5b183f80-8591-11eb-8619-847bab154531.png)

My question is how do i use a style sheet so I do not have to individually style a whole bunch of buttons?

**Comment by canton7** [comment]
Is this just a duplicate of #232?
